src/modules/notebook-panel/Overview.js

```diff
@@ -3,17 +3,22 @@ import {Alert, FlatList, Switch, Text, View} from 'react-native';
 
 import {Formik} from 'formik';
 import {Button} from 'react-native-elements';
-import {useDispatch, useSelector} from 'react-redux';
+import {connect, useDispatch, useSelector} from 'react-redux';
```

Comment:
@sdamaraju We've been getting rid of connects and just using useDispatch and useSelector